src/endpoint-configuration-service/src/main/java/org/fiware/sidecar/mapping/EndpointMapper.java

```diff
@@ -0,0 +1,62 @@
+package org.fiware.sidecar.mapping;
+
+import org.fiware.sidecar.model.AuthInfoVO;
+import org.fiware.sidecar.model.AuthType;
+import org.fiware.sidecar.model.AuthTypeVO;
+import org.fiware.sidecar.model.EndpointInfoVO;
+import org.fiware.sidecar.model.EndpointRegistrationVO;
+import org.fiware.sidecar.model.MustacheEndpoint;
+import org.fiware.sidecar.persistence.Endpoint;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+
+import java.util.Map;
+import java.util.UUID;
+
+@Mapper(componentModel = "jsr330")
```

Comment:
Probably worth adding a comment to link to jsr330

Reply:
Done